todo_app/tests/test_view_model.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+from todo_app.view_model import ViewModel
+from todo_app.data.item import Item
+
+@pytest.fixture
+def view_model():
+    items = [Item(1, "Write documentation" , "Done"), Item(2, "Program Function 1", "To Do"), Item(3, "Write unit tests", "Doing"),
+             Item(4, "Create requirements" , "Done"), Item(5, "Program Function 2", "To Do"), Item(6, "Write integration tests", "Doing")]
+    view_model = ViewModel(items)
+    return view_model
+
+correct_done_items = [Item(1, "Write documentation" , "Done"), Item(4, "Create requirements" , "Done")]
+correct_doing_items = [Item(3, "Write unit tests", "Doing"), Item(6, "Write integration tests", "Doing")]
+correct_todo_items = [Item(2, "Program Function 1", "To Do"), Item(5, "Program Function 2", "To Do")]
+
+@pytest.mark.parametrize("correct_done_item", correct_done_items)
+def test_view_model_done_property(view_model, correct_done_item):
+    assert correct_done_item in view_model.done_items
```

Comment:
Nice demo of parameterization here! I think it's worth also considering the other side of this assertion, i.e. this checks that all the `correct_done_items` are found after filtering, but it _doesn't_ check that nothing else comes, e.g. these tests would pass if each function just return everything

So potentially just extending with an assertion to check the right total number would make sense:
```suggestion
@pytest.mark.parametrize("correct_done_item", correct_done_items)
def test_view_model_done_property(view_model, correct_done_item):
    assert correct_done_item in view_model.done_items
    assert len(view_model.done_items) == len(correct_done_items)
```